### PR TITLE
Tim's First Pull Request OMG OMG

### PR DESCRIPTION
**Pull request by tim-pipi**
### Changes

Added "Team Members" section in the `README.md` file.

### CODE REVIEW

**Review by peasantbird — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -115,3 +115,8 @@ $ npm run prisma:reset
 #### Final steps
 
 1. Add the backend .env file - get it from an existing person using the repo.
+
+
+#### Team Members
+
+1. Timothy Putra Prasetio
```
Comment (m1oojv) [inline README.md:122]: Can u correct the styling to satisfy prettier linter

**Review by m1oojv — CHANGES_REQUESTED** [review]
lint error

**Comment by zsh-eng** [comment]
No need to clear the linting rules. Good job mr tim ! :D

**Pull request closed without merging**